i18n/ru/docusaurus-plugin-content-docs/current/branding.md

```diff
@@ -15,19 +15,19 @@ FSD прежде имел [другую легаси-айдентику](https:/
 
 :::
 
-## Название
+## Название {#title}
 
 - ✅ **Правильно:** `Feature-Sliced Design`, `FSD`
 - ❌ **Неправильно:** `Feature-Sliced`, `Feature Sliced`, `feature-sliced`, `feature sliced`, `FS`
 
-## Эмодзи
+## Эмодзи {#emojii}
 
 Ключевые концепции FSD хорошо отражаются в образе *"Торта"*.
 **Поэтому мы используем 🍰 как наше главное эмодзи, так же для более хороших ассоциаций**
```

Comment:
>Образ 🍰 торта хорошо отражает ключевые концепции FSD, поэтому он был выбран в качестве нашей "визитной карточки" среди emoji

>The cake 🍰 image represents FSD core concepts quite well, so it has been chosen as our signature emoji

I guess this would be enough 🤔

Reply:
fixed